model/database/common.yaml

```
@@ -61,7 +61,7 @@ groups:
       - ref: db.query.text
       - ref: db.query.summary
         requirement_level:
-          recommended: if readily available or if instrumentation supports query summarization.
+          recommended: if available or if the instrumentation supports generating a query summary.
```

if available seems maybe too generic

maybe?

```suggestion
          recommended: if available through instrumentation hooks or if the instrumentation supports generating a query summary.
```